tests/fitting/test_fitting_flows.py

```diff
@@ -409,11 +411,10 @@ def test_mlip_fit_maker_with_automated_separated_dataset(
     # Test if gap fit runs with pre_database_dir
     gapfit = MLIPFitMaker().make(
         species_list=["Li", "Cl"],
-        isolated_atoms_energy=[-0.28649227, -0.25638457],
```

Comment:
I am also wondering why this is removed here.

Reply:
see above

Reply:
gapfit takes the isolated atoms energy from the separate IsoAtom calculation entry in the xyz file directly.